oneview_redfish_toolkit/api/status_mapping.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2017) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+STATUS_MAP = {
+    "OK": {
+        "State": "Enabled",
+        "Health": "OK"
+    },
+    "Disabled": {
+        "State": "Disabled",
+        "Health": "OK"
```

Comment:
Is OK the right status here for Disabled? Should it be Warning or Critical?

Reply:
Fixed.